packages/cozy-sharing/src/components/useFetchDocumentPath.jsx

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect } from 'react'
+import { getFilesPaths } from '../helpers/files'
+
+export const useFetchDocumentPath = (client, document) => {
```

Comment:
imo `useFetch` is a bit weird for a name, as the hook user you don't really need to know it's doing a fetch behind the scenes.

Reply:
I think it's important to know that we're doing a fetch request when using this hook. It tells us if we want to pay the price of a request or not.